_layouts/default.html

```diff
@@ -21,21 +21,6 @@
             aria-label="Search text"
             placeholder="Type to search"
           />
-          <div class="clear-button">
```

Comment:
I would like to understand better why we're removing the clear search button without a replacement. I don't fully understand what overflowing the layout means.

Is it a functionality that can "just" be done without?

Reply:
We can add it back in, but it causes the sidebar to overflow currently, and I'm tired of fighting Docsify's defaults.

Reply:
And I don't think it provides much value.